hypothesis-python/docs/reference/django.rst

```diff
@@ -1,11 +1,11 @@
 .. _hypothesis-django:
 
-=====================
-Hypothesis for Django
-=====================
+======
+Django
+======
```

Comment:
I could be talked into keeping "Hypothesis for Django". I lean pretty minimalist in general (in life, designs, wording, etc), but I realize I can go too far sometimes.